lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_send_app_size_metrics.rb

```diff
@@ -0,0 +1,165 @@
+          the data to a endpoint over network. This can be useful e.g. to inspect the payload and debug it, or to store the metrics data as CI artefacts.
+        DETAILS
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :api_url,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_URL',
+            description: 'The endpoint API URL to publish metrics to. (Note: you can also point to a `file://` URL to write the payload to a file instead)',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :api_token,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_TOKEN',
+            description: 'The bearer token to call the API. Required, unless `api_url` is a `file://` URL',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :use_gzip_content_encoding,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_USE_GZIP_CONTENT_ENCODING',
+            description: 'Specify that we should use `Content-Encoding: gzip` and gzip the body when sending the request',
+            type: FastlaneCore::Boolean,
+            default_value: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_name,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_NAME',
+            description: 'The name of the app for which we are publishing metrics, to help filter by app in the dashboard',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_version,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_VERSION',
+            description: 'The version of the app for which we are publishing metrics, to help filter by version in the dashboard',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :build_type,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_BUILD_TYPE',
+            description: 'The build configuration for which we are publishing metrics, to help filter by build config in the dashboard. E.g. `Debug`, `Release`',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :source,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_SOURCE',
+            description: 'The type of event at the origin of that build, to help filter data in the dashboard. E.g. `pr`, `beta`, `final-release`',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :ipa_path,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_IPA_PATH',
+            description: 'The path to the .ipa to extract size information from',
```

Comment:
For consistency with other descriptions:

```suggestion
            description: 'The path to the `.ipa` to extract size information from',
```